packages/react/src/ButtonGroup/ButtonGroup.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import {ComponentMeta, ComponentStory} from '@storybook/react'
+import React from 'react'
+import {ButtonGroup} from '.'
+import {Button} from '../Button'
+
+export default {
+  title: 'Components/ButtonGroup',
+  component: ButtonGroup
+} as ComponentMeta<typeof ButtonGroup>
+
+const Template: ComponentStory<typeof ButtonGroup> = args => (
+  <ButtonGroup {...args}>
+    <Button>This is one button</Button>
+    <Button>This is the second button</Button>
+  </ButtonGroup>
+)
+
+export const Primary = Template.bind({})
+Primary.args = {}
+
+const SingleButtonTemplate: ComponentStory<typeof ButtonGroup> = args => (
+  <ButtonGroup {...args}>
+    <Button>This is one button</Button>
+  </ButtonGroup>
+)
+
+export const SingleButton = SingleButtonTemplate.bind({})
+
+const LargeButtonTemplate: ComponentStory<typeof ButtonGroup> = args => (
+  <ButtonGroup buttonSize="large" {...args}>
```

Comment:
Can this be set through args instead?

something like: 

```js
LargeButtonTemplate.args = {
  size: 'large'
}
```

I think the argTypes need to be manually configured too 👇 

https://user-images.githubusercontent.com/13340707/199083146-699275f6-f523-4e74-b213-34f4c8c59524.mov

Reply:
Unfortunately no because of the way the props are initialized on the copy. I was unable to use that specific prop name.